package.json

```diff
@@ -1,54 +1,78 @@
+  "version": "2.0.0-alpha",
+  "description": "Hawkular Charts components for Angular 4",
   "homepage": "https://github.com/hawkular/hawkular-charts",
-  "dependencies": {},
-  "devDependencies": {
-    "babel": "6.5.2",
-    "babel-core": "6.9.1",
-    "babel-preset-es2015": "6.9.0",
-    "bootstrap": "3.3.6",
-    "browser-sync": "^2.13.0",
-    "del": "2.2.0",
-    "event-stream": "3.3.3",
-    "express": "4.14.0",
-    "gulp": "3.9.1",
-    "gulp-angular-templatecache": "2.0.0",
-    "gulp-autoprefixer": "3.1.0",
-    "gulp-concat": "2.6.0",
-    "gulp-cssnano": "2.1.2",
-    "gulp-less": "3.1.0",
-    "gulp-notify": "2.2.0",
-    "gulp-rename": "1.2.2",
-    "gulp-size": "2.1.0",
-    "gulp-sourcemaps": "1.6.0",
-    "gulp-tslint": "5.0.0",
-    "gulp-typescript": "2.13.6",
-    "gulp-uglify": "1.5.3",
-    "gulp-util": "3.0.7",
-    "gulp-watch": "4.3.8",
-    "jquery": "^2.2.3",
-    "merge2": "1.0.2",
-    "patternfly": "3.5.0",
-    "run-sequence": "1.2.1",
-    "through2": "2.0.1",
-    "tslint": "3.11.0",
-    "underscore.string": "3.3.4",
-    "vinyl-map": "1.0.1",
-    "which": "1.2.10",
-    "wiredep": "4.0.0"
-  },
+  "main": "./bundles/hawkular-charts.umd.js",
+  "module": "./modules/hawkular-charts.es5.js",
+  "es2015": "./modules/hawkular-charts.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "node build.js",
+    "test": "karma start",
+    "pack-lib": "npm pack ./dist",
+    "publish-lib": "npm publish ./dist",
+    "compodoc": "compodoc -p tsconfig.json",
+    "compodoc-serve": "compodoc -s"
   },
+  "typings": "./index.d.ts",
   "repository": {
     "type": "git",
     "url": "https://github.com/hawkular/hawkular-charts.git"
   },
-  "engines": {
-    "node": ">=6.0.0"
+  "bugs": {
+    "url": "https://github.com/hawkular/hawkular-charts/issues"
+  },
+  "keywords": [
+    "hawkular",
+    "charts",
+    "angular"
+  ],
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "@angular/common": "^4.1.0",
+    "@angular/core": "^4.1.0",
+    "@angular/http": "^4.1.0"
+  },
+  "dependencies": {
+    "moment": "^2.9.0"
+  },
+  "devDependencies": {
+    "@angular/animations": "4.1.0",
+    "@angular/common": "4.1.0",
+    "@angular/compiler": "4.1.0",
+    "@angular/compiler-cli": "4.1.0",
+    "@angular/core": "4.1.0",
+    "@angular/http": "4.1.0",
+    "@angular/platform-browser": "4.1.0",
+    "@angular/platform-browser-dynamic": "4.1.0",
+    "@angular/platform-server": "4.1.0",
+    "@compodoc/compodoc": "1.0.0-beta.10",
+    "@types/jasmine": "2.5.52",
+    "@types/node": "8.0.1",
+    "chalk": "1.1.3",
+    "codelyzer": "3.1.1",
+    "core-js": "2.4.1",
+    "jasmine-core": "2.6.4",
+    "karma": "1.7.0",
+    "karma-chrome-launcher": "2.1.1",
+    "karma-jasmine": "1.1.0",
+    "karma-sourcemap-loader": "0.3.7",
+    "karma-spec-reporter": "0.0.31",
+    "karma-webpack": "2.0.3",
+    "reflect-metadata": "0.1.10",
+    "rollup": "0.43.0",
```

Comment:
I would prefer a more standard webpack build over rollup because the webpack plugins available are numerous and our other projects already use webpack.